### PR TITLE
FAQ: How can I fix 'browser was not found'?

### PR DESCRIPTION
**Pull request by psiinon**
What have I missed? I'm sure there will be something 😉 

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by kingthorin — COMMENTED** [review]
I'd suggest adding something about closing all browser(s), then opening the browser outside of ZAP (allowing it to apply any updates it might need).

I've seen Firefox in need of updating cause issues for browser launch on windows.

Otherwise this seems good to me.

**Comment by psiinon** [comment]
Oh yes - good point!

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+###  Check for Updates
+
+The first thing to do is to 'Check for Updates' and install any add-ons which are out of date, 
+in particular the webdriver and [Selenium](https://www.zaproxy.org/docs/desktop/addons/selenium/) ones.
+Browsers are updated very frequently and the updates often require corresponding changes to the webdrivers.
+If you dont have the right webdrivers then ZAP will not be able to launch the browsers.
+
+If you are on Windows then you may also need to restart ZAP. 
+Windows will refuse to allow ZAP copy the webdrivers to the correct location if they are currently being used.
```
Comment (kingthorin) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:22]:
```suggestion
Windows will refuse to allow ZAP to copy the webdrivers to the correct location if they are currently being used.
```

**Comment by psiinon** [comment]
Done 😁

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Thank you!

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+weight: 4
+---
+
+If you want to manually explore your target app then the easiest way is to launch your favourite browser
+from ZAP. ZAP will automatically configure it to proxy via ZAP and to ignore the certificate warnings
+you would otherwise get from the ZAP root CA certificate.
+
+But what can you do if ZAP fails to launch your browser?
+
+###  Check for Updates
```
Comment (kingthorin) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:14]:
```suggestion
### Check for Updates
```

**Comment by psiinon** [comment]
Updated based on comments from irc as well (thanks @thc202), and tweaked the "What is the default directory that ZAP uses?" FAQ

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**